plugins/announcements/package.json

```diff
@@ -63,6 +63,8 @@
     "@backstage/plugin-permission-react": "^0.4.26",
     "@backstage/plugin-search-common": "^1.2.14",
     "@backstage/plugin-search-react": "^1.8.0",
+    "@backstage/plugin-signals": "^0.0.10",
```

Comment:
Frotend plugin should not depend to another frontend plugin. This should be moved to `devDependencies`